packages/react/src/hooks/useSignIn.ts

```diff
@@ -0,0 +1,19 @@
+import { SignInResource } from '@clerk/types';
+
+import { useClientContext } from '../contexts/ClientContext';
+
+type UseSignInReturn =
+  | { isLoaded: false; signIn: null }
```

Comment:
❓ What's best between `null` , `undefined` or `[]` for the empty signIn object?

Reply:
Clerk.client is `undefined` before `load` runs, so I changed `null` to `undefined`. Does this look more consistent to you now?

Reply:
Yes